cli/pcluster/configure/easyconfig.py

```diff
@@ -116,49 +101,38 @@ def _get_subnets(conn, vpc_id):
 @handle_client_exception
 def _list_instances():  # Specifying the region does not make any difference
     """Return a list of all the supported instance at the moment by aws, independent by the region."""
-    return ec2_conn(DEFAULT_VALUES["aws_region_name"]).meta.service_model.shape_for("InstanceType").enum
+    return ec2_conn().meta.service_model.shape_for("InstanceType").enum
 
 
 def configure(args):
-    # Determine config file name based on args or default
-    config_file = (
-        args.config_file if args.config_file else os.path.expanduser(os.path.join("~", ".parallelcluster", "config"))
-    )
+    pcluster_config = PclusterConfig(config_file=args.config_file, file_sections=[AWS, GLOBAL, CLUSTER, ALIASES])
```

Comment:
It is a good idea but as long we support the AWS section i prefer to let the code as is.

Right now by default we are only reading the AWS section and it is useful because for `list_stacks`, `status`, `delete` commands we don't have to pass the `file_sections`,

If we parse all the sections by default we need to explicitly pass to these commands the `file_sections=[AWS]` to make sure to parse only that section.

But we are always parsing the AWS section so for the other commands we could have `file_sections=[GLOBAL, CLUSTER]`, the AWS section would be parsed anyway and the code would be inconsistent IMO.